Index: kmediaplayer-4.95.0/src/kmediaplayer/CMakeLists.txt
===================================================================
--- kmediaplayer-4.95.0.orig/src/kmediaplayer/CMakeLists.txt	2014-01-03 12:17:59.000000000 +0000
+++ kmediaplayer-4.95.0/src/kmediaplayer/CMakeLists.txt	2014-01-29 14:46:20.921350520 +0000
@@ -52,5 +52,6 @@
    DESTINATION ${SERVICETYPES_INSTALL_DIR})
 install(FILES
    org.kde.KMediaPlayer.xml
-   DESTINATION ${DBUS_INTERFACES_INSTALL_DIR})
+   DESTINATION ${DBUS_INTERFACES_INSTALL_DIR}
+   RENAME kf5_org.kde.KMediaPlayer.xml)
 
